addons/website_sale/views/templates.xml

```diff
@@ -2079,12 +2105,12 @@
                         <p>This is your current cart.</p>
                         <p>
                             <strong>
-                                <a t-attf-href="/shop/cart/?access_token=#{access_token}&amp;revive=squash">Click here</a>
+                                <a t-attf-href="/shop/cart/?id=#{id}&amp;access_token=#{access_token}&amp;revive=squash">Click here</a>
```

Comment:
{{id}} for IDEs

Reply:
What is the difference between `#{...}`and `{{...}}`? Why do you need that for IDE? Mine doesn't seems like it needs anything 🤔 
<img width="648" alt="Screenshot 2025-01-13 at 08 34 31" src="https://github.com/user-attachments/assets/376b3a13-f0de-4e7d-8302-bbaced0a0d0f" />

Reply:
The only documented format for format strings in QWeb is with double curly braces, so IDEs/plugins will naturally support only that one.

![image](https://github.com/user-attachments/assets/a20219bc-75e4-4ea9-911c-9bb635564b85)

It's not the end of the world if single curly braces are used, but it's not in line with the rest of the codebase and doesn't bring anything other than saving 2 chars.